gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -180,9 +180,9 @@ ext.externalDependency = [
     /**
      * Avoiding conflicts with Hive 1.x versions existed in the classpath
      */
-    "orcMapreduce":"org.apache.orc:orc-mapreduce:1.6.8-SNAPSHOT:nohive",
-    "orcCore": "org.apache.orc:orc-core:1.6.8-SNAPSHOT:nohive",
-    "orcTools":"org.apache.orc:orc-tools:1.6.8-SNAPSHOT",
+    "orcMapreduce":"org.apache.orc:orc-mapreduce:1.7.0-SNAPSHOT:nohive",
```

Comment:
Is this change necessary?

Reply:
The build was failing because v1.6.8 is not available in the snapshot repo here: https://repository.apache.org/content/repositories/snapshots/org/apache/orc/

Reply:
I think we should stay on 1.6.8 and no longer use the snapshot version, as we needed a patch from it before it was released but it looks like it was fully released May 21st. @sv2000

Reply:
@Will-Lo Thanks! Yes - it looks the changes backported to 1.6.8 are now fully released. We can drop the SNAPSHOT version and use the 1.6.8 released version.

Reply:
This should be done in a separate PR, I'll go open one today for the dependencies so this can stay clean.